lms/static/lms/js/require-config.js

```diff
@@ -49,7 +49,12 @@
             "backbone.paginator": "js/vendor/backbone.paginator.min",
             "underscore": "js/vendor/underscore-min",
             "underscore.string": "js/vendor/underscore.string.min",
-            "jquery": "js/vendor/jquery.min",
+            // The jquery-migrate library was added in upgrading from
+            // jQuery 1.7.x to 2.2.x.  This config allows developers
+            // to depend on "jquery" which opaquely requires both
+            // libraries.
+            "_jquery": "js/vendor/jquery.min",
```

Comment:
Actually, this might be fine if nobody ever requires _jquery. But it is probably better to follow the pattern we do with the other base_vendor files.